vhdbuilder/packer/install-dependencies.sh

```diff
@@ -102,6 +102,9 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
     containerd_version="1.4.4"
     downloadContainerd ${containerd_version}
     echo "  - [cached] containerd v${containerd_version}" >> ${VHD_LOGS_FILEPATH}
+    updated_containerd_version="1.5.5"
```

Comment:
nit: this doesn't really need to be a separate variable, or even a variable. but let's leave it as-is if you've already tested this.